### DISCUSSION TITLE
Is there a way to decode an expired JWT token?

### DISCUSSION THREAD

**Discussion by geoffroyp** [lexik/LexikJWTAuthenticationBundle#1022] [Q&A]
I'm facing a case where it would be very useful for me to be able to get my JWT token payload even when it is expired.
Right now, if I to do so, it throws a `JWTDecodeFailureException::EXPIRED_TOKEN`

I've found an old issue about it here mentioning this issue, but there the code snippet lacks of references (for instance, the "use" part)
https://github.com/lexik/LexikJWTAuthenticationBundle/issues/37

**Comment by FrenchGithubUser** [comment]
jwt tokens are juste base64 encoded, so you could use another function than the one from lexik
